src/test/java/com/decide/DecideTest.java

```diff
@@ -644,23 +644,39 @@ public void LIC12TestNegativeTwoPointPairs(){
     }
 
     @Test
+    /**
+     * Requirements: See `LIC5` documentation
+     * Contract:
+     *      Precondition: There exists a set of three points with the same 
```

Comment:
This precondition is probably inaccurate, specifically the phrasing "there exists a set".

> There exists a set of three points with the same coordinates, or there exists a set of three points with increasing consecutive x-values.

Should probably have something like:
> All sets of three points have the same coordinates or all sets of three points have decreasing x-values.

Reply:
I agree, should be fixed now!